.github/workflows/docker/Dockerfile-compilation-testing-ubuntu20.04

```diff
@@ -31,8 +31,14 @@ RUN [ ! -f sgx_linux_x64_sdk_$SGX_SDK_RELEASE_NUMBER.bin ] && \
     chmod +x sgx_linux_x64_sdk_$SGX_SDK_RELEASE_NUMBER.bin && echo -e 'no\n/opt/intel\n' | ./sgx_linux_x64_sdk_$SGX_SDK_RELEASE_NUMBER.bin && \
     rm -f sgx_linux_x64_sdk_$SGX_SDK_RELEASE_NUMBER.bin
 
-RUN echo "deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu bionic main" | tee /etc/apt/sources.list.d/intel-sgx.list && wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | apt-key add -
+RUN echo "deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu focal main" | tee /etc/apt/sources.list.d/intel-sgx.list && wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | apt-key add -
 
 RUN echo "deb http://cz.archive.ubuntu.com/ubuntu bionic main" >>  /etc/apt/sources.list && apt-get update && apt-get install -y libprotobuf10
 
 RUN apt-get update -y && apt-get install -y libsgx-dcap-quote-verify libsgx-dcap-quote-verify-dev libsgx-dcap-ql-dev libsgx-uae-service
+
+# install tdx
+RUN apt-get update && apt-get install -y pkgconf libprotobuf-c1 build-essential automake autoconf libtool 
+
+RUN apt-get install -y --no-install-recommends libsgx-urts libtdx-attest libtdx-attest-dev && \
+    ln -s /usr/lib/x86_64-linux-gnu/libtdx_attest.so.1.14.100.3 /usr/lib/libtdx_attest.so
```

Comment:
`/usr/lib/x86_64-linux-gnu/` is the standard path to lib. I think cmake file should be modified to support this.

Reply:
Please update the complication image(ubuntu 20.04 and Anolis 8.6) to make compilation check work.